addons/mod_tool/global/store.gd

```diff
@@ -53,17 +53,30 @@ func set_name_mod_dir(new_name_mod_dir: String) -> void:
 
 
 func init(store: Dictionary) -> void:
+	path_global_project_dir = ProjectSettings.globalize_path(ModLoaderUtils.get_local_folder_dir())
+	path_global_addon_dir = path_global_project_dir + "addons/mod_tool/"
+	if OS.has_feature("Windows"):
+		path_global_seven_zip = path_global_addon_dir + "vendor/7zip/windows/7zz.exe"
+	elif OS.has_feature("OSX"):
+		path_global_seven_zip = path_global_addon_dir + "vendor/7zip/mac/7zz"
+	elif OS.has_feature("X11"):
+		path_global_seven_zip = path_global_addon_dir + "vendor/7zip/linux/7zz"
+	else:
+		printerr("OS currently not supported to export zips via mod tool. Please open an issue on GitHub")
+
+	if not File.new().file_exists(path_global_seven_zip):
+		printerr("7zip installation not found at path %s. Please install if at that location." % path_global_seven_zip)
```

Comment:
`printerr()` or `ModToolUtils.output_error()`? 
Can't remember if we remove the output anyway when we move to the main scene view.
But I think we can keep `output_error()` and just do the `printerr()` there?

Reply:
yep, good point